### DISCUSSION TITLE
Disabling keyboard prediction/acknowledgement on ssh sessions

### DISCUSSION THREAD

**Discussion by dramborleg** [wezterm/wezterm#1595] [Q&A] [answered]
Not sure what to call this, but it seems that on ssh connections wezterm predicts what would be displayed based on what is typed. There seems to be an intermediate state where the character is printed in a different color before it is acknowledged/confirmed by the ssh session (then turns to normal/expected color). It makes the characters flash a lot and also makes the characters of passwords (eg when using sudo) temporarily visible so I'd like to disable it. Most other terminals seem to print nothing until the remote session responds, which means you can type ahead of the cursor and it doesn't keep up with the keyboard. Is there an option that can be tweaked for this?

**Comment by wez** [comment, marked as answer]
Assuming that you are using the multiplexing feature, you may be interested to follow along with: https://github.com/wez/wezterm/pull/1518

**Reply by dramborleg** [reply]
Thanks looks like what I was asking for!
